src/App.tsx

```diff
@@ -1,27 +1,59 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { getAll, get5First, getRed } from './api/goods';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+  const handleClickGetAll = async () => {
+    const allGoods = await getAll();
```

Comment:
Hi! It would be better to create one common handler for all onClick, and pass functions of request into the parametr as callbacks:
`
const commonHandler = (callback) => {
  const requestedGoods = await callback();
  setGoods(requestedGoods);
};
...
onClick = {commonHandler(getAll())}
onClick = {commonHandler(get5First())}
onClick = {commonHandler(getRedGoods())}
`